TestApp/copy-code-changes-to-git.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 echo 'Copy SDK code changes made inside node_modules to git...'
 cd `dirname $0`
-for i in '' '-analytics' '-crashes' '-push'
+for i in '' '-analytics' '-crashes' '-push' '-data'
```

Comment:
Looks like we forgot `-auth`. Let's add it.

Reply:
`-auth` was added in another [PR](https://github.com/microsoft/appcenter-sdk-react-native/pull/638/files#diff-7dd8fda13620c96a2fc8ad5978c44903L4). It’s in the develop branch: [https://github.com/microsoft/appcenter-sdk-react-native/blob/25163fe18224dc502547c594bb3ccaeb0784802d/TestApp/copy-code-changes-to-git.sh#L4](https://github.com/microsoft/appcenter-sdk-react-native/blob/25163fe18224dc502547c594bb3ccaeb0784802d/TestApp/copy-code-changes-to-git.sh#L4)

Reply:
Got it, thanks.